### PR TITLE
fix(jira-linked): add Jenkins bot exception

### PR DESCRIPTION
**Pull request by pmlanger**
Checks in event-dispatcher fail on `main`, because the "Jenkins User" bot is not part of the list of exceptions: https://github.com/Typeform/event-dispatcher/commit/1778094012fd4c5b2e399d4f7421e0c3c964c16c
Therefore, adding it to the list of exceptions for that check.

### CODE REVIEW

File: src/conditions/triggeredByBot.ts [hunk 1]
```diff
@@ -9,7 +9,9 @@ export const BOT_USERS = [
   'dependabot-preview[bot]',
   'tf-security',
   'seti-tf',
-  'tf-IT'
+  'tf-seti',
```
Comment (jhuliano) [inline src/conditions/triggeredByBot.ts:12]: looks like this is duplicated, perhaps should we remove one of them?

Comment (pmlanger) [inline src/conditions/triggeredByBot.ts:12]:
```suggestion
```

Comment (pmlanger) [reply src/conditions/triggeredByBot.ts]: I simply removed that change now. I am not very sure that this actually filters all the tf-seti bots as it is now, as their author name I have seen it as "Engineering Intelligence", but I guess that's another thing to tackle (rename the bots for example).

**Review by jhuliano — APPROVED** [review]

**Pull request merged**